packages/celotool/src/lib/helm_deploy.ts

```diff
@@ -530,6 +530,7 @@ async function helmParameters(celoEnv: string) {
     `--set geth.backup.enabled=${fetchEnv(envVar.GETH_NODES_BACKUP_CRONJOB_ENABLED)}`,
     `--set bootnode.image.repository=${fetchEnv('GETH_BOOTNODE_DOCKER_IMAGE_REPOSITORY')}`,
     `--set bootnode.image.tag=${fetchEnv('GETH_BOOTNODE_DOCKER_IMAGE_TAG')}`,
+    `--set bootnode.internal=${fetchEnv(envVar.INTERNAL_BOOTNODE)}`,
```

Comment:
be good to consider a default for this via `fetchEnvOrFallback`. I would say the default should be `true` as most networks we deploy it'd be good to save the LB quota.